### ISSUE TITLE
Pick up wallet updates made externally

### ISSUE THREAD

**Issue by barnabee** [vegaprotocol/vegawallet-desktop#288]
When we are creating a wallet/ keys in the CLI, the desktop wallet should be able to identify this and update accordingly. 

It may make sense to do this after the core work to minimise the need for repeatedly entering a passphrase. But will need to be done before alpha mainnet as it will introduce a breaking change to the APIs that we seek to avoid once in alpha mainnet

# Software information
Build info:
- Version: v0.2.1
- Commit hash: unknown

Installation:
- [x] I downloaded it from the GitHub releases page.

# Problem encountered / observed behaviour
After adding keys via the CLI wallet app they did not appear in the desktop wallet until I quit and reloaded it.

# Expected behaviour
Newly generated keys (and indeed other changes) are detected and shown in the desktop wallet.

# Steps to reproduce

## Manual
1. Open desktop wallet
2. Unlock a wallet "W" and see a key/keys
3. Use the CLI wallet to generate an additional key for wallet "W"
4. Check desktop wallet, only the keys that were already visible in step 2 are shown, whereas the newly generated keys would be expected to be visible
5. Quite desktop wallet
6. Open desktop wallet again
7. Unlock wallet "W"
8. Now the new key(s) are shown

# Additional context

- This is probable low priority
- It is not clear to me if there is a risk of corruption/data loss if, after doing a CLI update that was not detected in the desktop wallet, a change was also made to the same wallet via the desktop wallet GUI, but if so it may be a more serious bug and at least require the desktop wallet and CLI wallet and service to lock the wallet file(s) while they are open and prevent both being used at once…

**Comment by ValentinTrinque** [comment]
We would need to watch a file for changes and warn the front-end of any changes. That is something that is worth adding on the JSON-API v2 I guess.
